### PR TITLE
docker for using debian builds of grpc

### PR DESCRIPTION
**Pull request by nfultz**
@daroczig

### CODE REVIEW

File: inst/docker_standard/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+## Install grpc and protobuf from deb
+RUN apt-get update && apt-get install -y \
+  libprotoc-dev libprotobuf-dev protobuf-compiler \
+  libgrpc3 libgrpc-dev && \
+  apt-get clean && rm -rf /var/lib/apt/lists/
+
+
+
+## set locale for R
```
Comment (daroczig) [inline inst/docker_standard/Dockerfile]: `rocker` already does this

File: inst/docker_standard/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+RUN echo "en_US.UTF-8 UTF-8" >> /etc/locale.gen \
+	&& locale-gen en_US.utf8 \
+	&& /usr/sbin/update-locale LANG=en_US.UTF-8
+ENV LC_ALL en_US.UTF-8
+ENV LANG en_US.UTF-8
+
+## install R packages
+RUN echo 'options(repos = c(CRAN = "https://cran.rstudio.com/"), download.file.method = "libcurl")' >> /etc/R/Rprofile.site && \
+  Rscript -e "install.packages(c('devtools', 'testthat', 'processx', 'RProtoBuf'))" && \
```
Comment (daroczig) [inline inst/docker_standard/Dockerfile]: `install2.r` comes with `rocker`

File: inst/docker_standard/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  Rscript -e "devtools::install_github('nfultz/grpc')" && \
```
Comment (daroczig) [inline inst/docker_standard/Dockerfile]: `installGithub.r` comes with `rocker`

File: inst/docker_standard/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+  libgflags-dev libgtest-dev clang libc++-dev \
+  unzip git curl wget \
+  pkg-config locales  \
+  libssl-dev libcurl4-openssl-dev libxml2-dev libfftw3-dev && \
+  apt-get clean && rm -rf /var/lib/apt/lists/
+
+## Install grpc and protobuf from deb
+RUN apt-get update && apt-get install -y \
+  libprotoc-dev libprotobuf-dev protobuf-compiler \
+  libgrpc3 libgrpc-dev && \
```
Comment (daroczig) [inline inst/docker_standard/Dockerfile]: this is a bit outdated version but I'm fine with it

File: inst/docker_standard/Dockerfile [hunk 5]
```diff
@@ -0,0 +1,37 @@
+FROM rocker/r-base
+MAINTAINER Neal Fultz <nfultz@gmail.com>
+
+ENV GRPC_RELEASE_TAG v1.4.5
```
Comment (daroczig) [inline inst/docker_standard/Dockerfile]: no need for these if we install grpc from debian apt repos

**Comment by daroczig** [comment]
If we want to stick with the slightly outdated debian apt packages and not sticking with our custom defined versions, I'm totally fine with this -- left some suggestions how to simplify the dockerfile. Note: I did not manage to build grpc from source on debian due to openssl 1.1 as mentioned at https://github.com/nfultz/grpc/pull/7#discussion_r143340644 and c49918731b6180361e7876f661fbf16ad4611e35

**Comment by nfultz** [comment]
Theres some value in having both a "stable" build and a "dev" build, I'd rather maintain a docker file than `install.md`.

**Comment by nfultz** [comment]
Cleaned up docker_stable per your recommendations, next I'll set up a dockerhub acct and wire it up.

**Comment by nfultz** [comment]
OK, I have the dockerhub set up, I think, so I'm going to merge this in now.

**Comment by nfultz** [comment]
https://hub.docker.com/r/rgrpc/grpc_dev/

**Comment by daroczig** [comment]
Very cool :+1: 
Let me know if I can help with anything here.

**Pull request merged**